dune-project

```diff
@@ -0,0 +1,3 @@
+(lang dune 1.3)
```

Comment:
Are we using any 1.3 features here? Seems like we're not, so 1.0 should be good enough.

Reply:
I'll take your word for it, suppose you should know ;-)